application/helpers/remotecontrol/remotecontrol_handle.php

```diff
@@ -1673,26 +1673,32 @@ public function get_question_properties($sSessionKey, $iQuestionID, $aQuestionSe
                             $aResult[$sPropertyName] = 'No available answers';
                         }
                     } elseif ($sPropertyName == 'attributes') {
-                        $oAttributes = QuestionAttribute::model()->findAllByAttributes(array('qid' => $iQuestionID, 'language' => null), array('order' => 'attribute'));
-                        if (count($oAttributes) > 0) {
-                            $aData = array();
-                            foreach ($oAttributes as $oAttribute) {
-                                $aData[$oAttribute['attribute']] = $oAttribute['value'];
+                        $questionAttributeHelper = new LimeSurvey\Models\Services\QuestionAttributeHelper();
+                        $questionAttributes = $questionAttributeHelper->getQuestionAttributesWithValues($oQuestion);
```

Comment:
`advancedOnly` must be true because 

1. same in 3.X : attributes return only advanced attributes 

2. Seems it's a broken function

Reply:
> > 2. Seems it's a broken function
> 
> What is broken?

Issue with attribute[gid] = "" and attribute[preg] = "", but gid=124 and pref="string" 
Seems to retune null or false.